dotnet/samples/KernelSyntaxExamples/Example02_Plugin.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.ComponentModel;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.ChatCompletion;
+using Microsoft.SemanticKernel.Connectors.OpenAI;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Examples;
+
+/// <summary>
+/// This example shows how to create a plugin class and interact with as described at
+/// https://learn.microsoft.com/en-us/semantic-kernel/overview/
+/// This sample uses function calling, so it only works on models newer than 0613.
+/// </summary>
+public class Example02_Plugin : BaseTest
```

Comment:
We have started to move away from the `ExampleNN_<Description>` naming convention so we should not follow the pattern. The pattern used by the Getting Started samples is the one we want to follow.

That said, I wonder do we want to add these to the Kernel Syntax Examples. I thought we where just going to move the documentation sample projects as is to this repo and then make sure they were being built and tested.

Adding them to the Kernel Syntax Examples could make them harder to consume by someone following the documentation.

@matthewbolanos What are your thoughts here?

Reply:
Moving these examples to a standalone project